markdown/org/showcase/sleeveless-simon-by-lasermonkey12/en.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: "Sleeveless Simon by lasermonkey12"
-caption: "A sleeveless Simon shirt"
+title: "Sleeveless Simone by lasermonkey12"
+caption: "A sleeveless Simone shirt"
 date: 20240106
 intro: "Maker lasermonkey12 has made lots of great Simon shirts, including this sleeveless one."
-designs: ["simon"]
-maker: Natalia
+designs: ["simone"]
+maker: lasermonkey12
+author: 22007
 ---
 
 Maker lasermonkey12 has made lots of great Simon shirts, including this sleeveless one. We love the fresh look, well-suited for hot weather! This was shared on [Discord](https://discord.freesewing.org/) and is reposted here with permission.
```

Comment:
Should also say Simone here, then?

Reply:
Yes, please go ahead and make the change.

Reply:
I don't think I have permissions to edit stuff in your pull request. :frowning_face:

Reply:
Ah, bad luck. I fixed it.